.github/workflows/publish-packager-nodejs.yml

```diff
@@ -0,0 +1,403 @@
+name: CI
+
+env:
+  DEBUG: napi:*
+  APP_NAME: packager
+  MACOSX_DEPLOYMENT_TARGET: "10.13"
+
+permissions:
+  contents: write
+  id-token: write
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseId:
+        description: "ID of the `@crabnebula/packager` release"
+        required: true
+  repository_dispatch:
```

Comment:
we need to update covector-publish-or-version.yml as well

Reply:
I didn't do it yet because we have that issue on Tauri.. we didn't find a way to parse the output yet

Reply:
fck, I will create a repo to test again but anyways let's add those changes as well and fix it later in another PR

Reply:
I believe I found a fix for it that shouldn't break again https://github.com/amr-crabnebula/covector-test/actions/runs/6896036131/job/18761297111#step:6:1

Reply:
basically use inline-js
```yml
      - uses: actions/github-script@v6
        id: set-result
        with:
          result-encoding: string
          script: |
            const output = `${{ toJSON(steps.covector.outputs) }}`;
            const [_, id] = /"-org-dashed-pkg-releaseId": "([0-9]+)"/g.exec(output);
            return id;

      - name: Get result
        run: echo "${{steps.set-result.outputs.result}}"
```